receiver/githubreceiver/internal/scraper/githubscraper/schema.graphql

```diff
@@ -1443,7 +1443,7 @@ type CommitComment implements Node & Comment & Deletable & Minimizable & Updatab
   viewerCanReact: Boolean!
   "Check if the current viewer can update this object."
   viewerCanUpdate: Boolean!
-  "Reasons why the current viewer can not update this comment."
+  "Reasons why the current viewer cannot update this comment."
```

Comment:
Hm I believe this is a generated file, but there is no header on the file saying so. @adrielp can you confirm?

I think if the file is autogenerated then we don't want to modify it?

Reply:
anyway, it's omitted in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37606